snuba/datasets/cdc/message_filters.py

```diff
@@ -1,32 +0,0 @@
-from arroyo import Message
-from arroyo.backends.kafka import KafkaPayload
-
-from snuba.datasets.message_filters import StreamMessageFilter
-
-KAFKA_ONLY_PARTITION = (
-    0  # CDC only works with single partition topics. So partition must be 0
-)
-
-
-class CdcTableNameMessageFilter(StreamMessageFilter[KafkaPayload]):
```

Comment:
so this file is just dead code?

Reply:
The code in this file was consolidated to `snuba/datasets/message_filters.py` with the rest of the filters.